packages/lang-server/src/location.ts

```diff
@@ -48,6 +49,52 @@ export const getLocations = async (
   (await workspace.getSourceRanges(ElemID.fromFullName(fullname)))
     .map(range => ({ fullname, filename: range.filename, range }))
 
+const extractStaticFileAttributes = (element: Element, refPath: string[]):
+{ staticFile: unknown
+  fullname: string } | undefined => {
+  if (isInstanceElement(element)) {
+    const staticFile = _.get(element.value, refPath)
+    const fullname = element.elemID.createNestedID(...refPath).getFullName()
+    return { staticFile, fullname }
+  }
+  if (isObjectType(element) || isField(element)) {
+    const staticFile = _.get(element.annotations, refPath)
+    const fullname = element.elemID.createNestedID('annotation', ...refPath).getFullName()
+    return { staticFile, fullname }
+  }
+  return undefined
+}
+
+export const getStaticLocations = (element: Element, refPath: string[],
+  token: string): SaltoElemLocation | undefined => {
```

Comment:
line break style convention nit
```suggestion
export const getStaticLocations = (
  element: Element,
  refPath: string[],
  token: string,
): SaltoElemLocation | undefined => {
```